src/GuiBase/BaseApplication.hpp

```diff
@@ -175,6 +191,12 @@ class RA_GUIBASE_API BaseApplication : public QApplication
     bool m_recordGraph;
```

Comment:
When editing files that lacks doc and inline intialization, just add what is simple and quick.

Reply:
Don't know what this bool is for, help appreciated.

Reply:
Done. I' commit the comment.